### DISCUSSION TITLE
A bug in achivements where you arent able to enable your achivements if you disable all of them unless you get new ones.

### DISCUSSION THREAD

**Discussion by RobiMez** [community/community#118510] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

If you turn off the visibility of all your achievements one by one using the eye icon on the achievements tab , you aren't able to enable them back because the section would not be rendered, and you cant also re-enable them from the profile settings either.

![image](https://github.com/community/community/assets/53259730/705be7bc-2a77-4f44-83ab-4d243441028c)
![image](https://github.com/community/community/assets/53259730/e7b740a2-9054-42b6-ae0a-47670266ea83)
![image](https://github.com/community/community/assets/53259730/12e0eecb-36eb-4a77-abd7-d6cd09b11a2c)
![image](https://github.com/community/community/assets/53259730/a1677c63-3fc1-41c8-ab76-bda5b1f6c966)

**Comment by jfullstackdev** [comment]
![image](https://github.com/community/community/assets/47092464/9a0ad8ee-1c43-4ee1-9878-1c5f7cfb19b4)

![image](https://github.com/community/community/assets/47092464/2a07caba-fd1b-4f85-8f0c-ccff8a7bb3a7)

I can easily switch as you can see

**Reply by RobiMez** [reply]
switch all of them off 
close the tab 
then switch them back on from your profile page

**Reply by RobiMez** [reply]
I know you can turn individual badges on and off , the section for doing that ( the achievements section ) is inaccessible if you have all of them off ( individually , not from the settings page ) is what i'm saying.

**Reply by jfullstackdev** [reply, marked as answer]
yes, but you must remember the URL, though again, I will consider this as bug as not all don't know what to do in this situation, this is the URL pattern

https://github.com/[username]?tab=achievements

**Reply by RobiMez** [reply]
Yup the URL trick works , i didn't know what the URL was though so i had to ask a few of my friends to star my repo to get the starstruck achievement. 
Yeah tho , maybe we can add a link to the URL on the settings page next to the show achievements on my profile checkbox. 
i don't know if the GitHub page itself is open-sourced but it would be super if i made a pr fixing this 😄.

![image](https://github.com/community/community/assets/53259730/747cbc0f-6c6f-4101-8937-f8ff90da6ff1)

**Reply by jfullstackdev** [reply]
it's now up for the GitHub Staff to implement such if they want to do so,

**Reply by RobiMez** [reply]
Nice , thanks !
